articles/data-share/supported-data-stores.md

```diff
@@ -51,6 +51,8 @@ Please refer to [Share and receive data from Azure Blob Storage and Azure Data L
 ## Share from a SQL-based source
 Azure Data Share supports sharing of both tables and views from Azure SQL Database and Azure Synapse Analytics (formerly Azure SQL DW), and sharing of tables from Azure Synapse Analytics (workspace) dedicated SQL pool. Sharing from Azure Synapse Analytics (workspace) serverless SQL pool is not currently supported. Data consumers can choose to accept the data into Azure Data Lake Storage Gen2 or Azure Blob Storage as csv or parquet file, as well as into Azure SQL Database and Azure Synapse Analytics as tables.
 
+[!NOTE] CSV file format does not support quoteChar. So there won't be any single character that would quote column values if it contains column delimiter **comma** `,`.
```

Comment:
Our response will be delayed due to holidays this week. Waiting for a technical review by the doc owner.

Please format the Note like this:
```suggestion
> [!NOTE] 
> The CSV file format does not support quotechar. So there won't be any single character that would quote column values if it contains column delimiter **comma** `,`.
```